src/scripts/utils.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python
+import os
+from constants import _constants
+
+def _write_file(settings, content):
+    file_count = 1
+    output_dir = os.path.abspath(settings._OUTPUT_DIR)
+    if not os.path.isdir(output_dir):
+        os.makedirs(output_dir)
+    while True:
+        filename = settings._GENERATED_FILENAME.format(NUMBER = file_count)
+        file_to_open = os.path.join(output_dir, filename)
+        with open(file_to_open, "w") as file:
+            file.write("%s\n" % (settings._FILE_HEADER))
+            counter = settings._TEST_CASE_IN_A_FILE
+            for i in range(len(content)):
+                file.write("%s" %(content[0]))
+                del content[0]
+                counter -= 1
```

Comment:
What is the reason of removing elements from the list? Cannot we just simply iterate through the list elements?